site/profile/manifests/app/rgbank/db.pp

```diff
@@ -1,5 +1,31 @@
-class profile::app::rgbank::db {
+# rgbank database profile
+class profile::app::rgbank::db (
+  $split = false,
+) {
 
-  include ::profile::app::db::mysql::server
+  include ::profile::platform::baseline
```

Comment:
Im not sure we want to include this at the app profile level, and I tend to think it belongs at the role:: level.  Thinking being if we renamed, moved, refactored - there is only 1 level of code to update.  Thoughts?

Reply:
@murdok5 Totally overlooked this thread. I'm not sure we should be keeping the rgbank module / profile at all. Do you know who are still using it?